pkg/util/annotation_test.go

```diff
@@ -425,3 +466,223 @@ func TestMergeAnnotation(t *testing.T) {
 		})
 	}
 }
+
+func TestDedupeAndMergeAnnotations(t *testing.T) {
+	tests := []struct {
+		name               string
+		existAnnotation    map[string]string
+		newAnnotation      map[string]string
+		expectedAnnotation map[string]string
+	}{
+		{
+			name:               "Nil both Annotation",
```

Comment:
The case name can be named in the same style as the existing case. For example: `nil both annotation.`